### ISSUE TITLE
Fix Ctrl-V on Unix

### ISSUE THREAD

**Issue by gwenn** [kkawakam/rustyline#65]

**Comment by legolord208** [comment]
I think you meant Ctrl+Shift+V...

**Comment by gwenn** [comment]
I mean:
```sh
$ bind -p | grep "quoted-insert"
"\C-q": quoted-insert
"\C-v": quoted-insert
```

**Comment by legolord208** [comment]
Oh! I never knew those were supposed to work :stuck_out_tongue:

**Comment by gwenn** [comment]
Still buggy in 2.0 (for example with a tab)

**Comment by gwenn** [comment]
Fixed by 4.0 release.
